### PR TITLE
Updating email templates with better language

### PR DESCRIPTION
**Pull request by jasonamyers**
Signed-off-by: Jason Myers <jason@jasonamyers.com>

### CODE REVIEW

File: templates/email/accept.txt [hunk 1]
```diff
@@ -4,69 +4,50 @@ Congratulations! Your talk,
...
-please apply for financial aid.
-
-More information about financial aid can be found at
-https://us.pycon.org/2018/financial-assistance/
-
-If you're applying for financial aid, make sure to mention that you're
-an accepted speaker.  Please contact the financial aid team at
-pycon-aid@python.org if you have any questions.
+PyCon provides speaker financial assistance to anyone that requests it. If you
+would like to get speaker financial assistance, please select the box on your
```
Comment (ewdurbin) [inline templates/email/accept.txt]: I believe we also need to ask them to fill out a finaid app so we know what assistance they require.

File: templates/email/decline.txt [hunk 2]
```diff
@@ -31,21 +39,23 @@ the PyCon website, https://us.pycon.org/2018/events/open-spaces/
...
-free to contact me (jason+pcc@mailthemyers.com), or the conference chair,
-Ernest Dubin (ernest@python.org).
+free to contact the Program Committee Chairs (pycon-program-committee@python.org).
 
 On behalf of the PyCon staff, I want to thank you again for your
 submission. You made our job of picking talks very hard, and the
 conference will be better for it.
 
-I hope to see you in Portland!
+I hope to see you in Cleveland!
```
Comment (ewdurbin) [inline templates/email/decline.txt:51]: 🎉

**Review by ewdurbin — COMMENTED** [review]
@jasonamyers I think we’re good, but definitely need to determine what specific needs speakers need beyond just checking the box, and need to know that before finaid deadline.

File: templates/email/accept.txt (hunk 1 above)
Comment (jasonamyers) [reply templates/email/accept.txt]: is that the official answer, because Ewa had commented on the google docs about using the checkbox on speaker profile?

Comment (jasonamyers) [reply templates/email/accept.txt]: Started an email chain off of github to work towards the answer.

**Review by ewdurbin — APPROVED** [review]

**Comment by ewdurbin** [comment]
@ewajodlowska any additional feedback?

**Comment by ewdurbin** [comment]
@ejodlowska any final thoughts or suggestions?

**Comment by ejodlowska** [comment]
LGTM.

**Pull request merged**